libraries/AP_HAL_PX4/RCOutput.cpp

```diff
@@ -366,16 +358,7 @@ uint16_t PX4RCOutput::read(uint8_t ch)
     if (ch >= PX4_NUM_OUTPUT_CHANNELS) {
         return 0;
     }
-    // if px4io has given us a value for this channel use that,
-    // otherwise use the value we last sent. This makes it easier to
-    // observe the behaviour of failsafe in px4io
-    for (uint8_t i=0; i<ORB_MULTI_MAX_INSTANCES; i++) {
-        if (_outputs[i].pwm_sub >= 0 && 
-            ch < _outputs[i].outputs.noutputs &&
-            _outputs[i].outputs.output[ch] > 0) {
-            return _outputs[i].outputs.output[ch];
```

Comment:
To my understanding this has to do with following:

https://github.com/ArduPilot/ardupilot/pull/6032/commits/26eb5a10380d73f52e87ae7bd22da6b3e9bacaaa#diff-2881b03c35e023919c47afed5450cc42L573

Which is not needed.